src/ray/gcs/store_client/redis_store_client.cc

```diff
@@ -268,36 +352,31 @@ Status RedisStoreClient::DoPut(const std::string &key,
           callback(added_num != 0);
         };
   }
-
-  auto shard_context = redis_client_->GetShardContext(key);
-  return shard_context->RunArgvAsync(args, write_callback);
+  SendRedisCmd({key}, std::move(args), std::move(write_callback));
+  return Status::OK();
 }
 
 Status RedisStoreClient::DeleteByKeys(const std::vector<std::string> &keys,
```

Comment:
do we have test cover delete?

Reply:
We have tests for delete in CI (single or multi)